tests/lib/nm/route_test.py

```diff
@@ -59,6 +64,49 @@
     ids=['ipv4', 'ipv6'])
 
 
+# This function should be above @parametrize_ip_ver_routes_gw due to some
```

Comment:
It is not pytest, but python. The `parametrize` is called on module import (as it is global), therefore it calls at import this function. It can call/reference it only if the function got loaded, therefore it needs to be placed before.
(it can be solved but I have no problem with this as is)
I think there is no need for this comment.

Reply:
I added that line just because you require private function be placed after the caller.
Anyway, comments removed.